Applications/CLI/ogs.cpp

```diff
@@ -99,6 +100,14 @@ int main(int argc, char *argv[])
 		"output directory");
 	cmd.add(outdir_arg);
 
+	TCLAP::ValueArg<std::string> log_level_arg(
+		"l", "log-level",
```

Comment:
I like the shortcut, I propose to eventually change if it clashes with some other parameter in the future. Btw. OGS will never be feature-complete...